napari/layers/_tests/test_layer_actions.py

```diff
@@ -3,6 +3,7 @@
 import numpy as np
 import pytest
 
+from napari._qt.widgets.qt_action_context_menu import QtActionContextMenu
```

Comment:
This make the headless test fail as `napari._qt` can't be imported.

Reply:
Oh :(, I'm going to take a look if there is a better way to query the actions. Thanks @Carreau

Reply:
@Carreu the `make_napari_viewer` also crashes the headless tests, and this is necessary to test this. Should I move this test to another directory?

Reply:
you don't need a full Qt-ready viewer here right?  Just use `ViewerModel()`... ?